osmnx/core.py

```diff
@@ -503,15 +503,23 @@ def gdf_from_places(queries, gdf_name='unnamed', buffer_dist=None):
         later)
     buffer_dist : float
         distance to buffer around the place geometry, in meters
+    which_results : list
+        list (with the same size of queries) of max number of results to return
```

Comment:
if not None, a list of max number of results to return and which to process upon receipt, for each query in queries

Reply:
Done